pkg/cache/azure_cache.go

```diff
@@ -21,6 +21,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/mohae/deepcopy"
```

Comment:
it is not actively maintained, could you add a new deep copy utils instead?

Reply:
How about this lib? It supports deepcopy and has many users.
https://github.com/jinzhu/copier

Reply:
@feiskyer I spent some time debugging with this new lib and realized that it has a bug about nested struct deep copy. And the fix PR is still pending. https://github.com/jinzhu/copier/pull/157
So I decided to use the first lib and put the deepcopy.go into our lib, not in vendor. PTAL. Also please check if there's any copyright issue. Thanks.